### PR TITLE
Update F_GetArmorType()

### PR DESCRIPTION
**Pull request by Emistry**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 
- [x] Master

**Issues addressed:**
Update missing equip types.

[//]: # (Issue Tracker Number if any.)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/1836)

File: npc/other/Global_Functions.txt [hunk 1]
```diff
@@ -269,19 +269,33 @@ function	script	F_GetWeaponType	{
...
+		case 131072: return "Shadow Weapon"; break;
+		case 262144: return "Shadow Shield"; break;
+		case 524288: return "Shadow Shoes"; break;
+		case 1048576: return "Shadow Accessory"; break;
+		case 2097152: return "Shadow Accessory"; break;
+		case 3145728: return "Shadow Accessory"; break;
+		default: return "Unknown Equip Type"; break;
 	}
 	end;
+	end;
```
Comment (dastgir) [inline npc/other/Global_Functions.txt]: Multiple end?

Comment (Emistry) [reply npc/other/Global_Functions.txt]: ops... fixed.

**Comment by dastgir** [comment]
Do case statement accept hex value?

**Comment by Emistry** [comment]
not sure if anyone does using hex value here... even in-game we need to use the `axtoi` to convert it to decimal before we could use it, aren't?

most of the time they just use the `getiteminfo(id, 5)` to retrieve the value.

**Comment by MishimaHaruna** [comment]
@dastgir yes, it can take any integer value (decimal, hexadecimal, etc) as well as any constant

@Emistry `axtoi()` is only needed when you have the hex value in a string (because the automatic conversion only looks for decimal values)

**Pull request merged**